src/ivoc/ivocmain.cpp

```diff
@@ -372,6 +374,8 @@ int ivocmain(int argc, const char** argv, const char** env) {
  * \return 0 on success, otherwise error code.
  */
 int ivocmain_session(int argc, const char** argv, const char** env, int start_session) {
+    nrn::Instrumentor::init_profile();
```

Comment:
Just out of curiosity: these entry points were not done correctly?

Reply:
Also
```
$ grep -R init_profile src 
src/coreneuron/apps/coreneuron.cpp:    coreneuron::Instrumentor::init_profile();
src/coreneuron/utils/profile/profiler_interface.h:    /*! \fn init_profile
...
src/coreneuron/utils/profile/profiler_interface.h:    detail::InstrumentorImpl::init_profile();
src/utils/profile/profiler_interface.h:    /*! \fn init_profile
...
src/utils/profile/profiler_interface.h:    detail::InstrumentorImpl::init_profile();
```
shows it's only set for coreneuron, not neuron.

Reply:
thanks for clarifying!